### DISCUSSION TITLE
Who is making command work as alt?

### DISCUSSION THREAD

**Discussion by marlun** [kovidgoyal/kitty#5709] [Q&A] [answered]
When I use tmux inside Kitty the CMD keys seems to work like ALT because I can do CMD+. to get the last parameter in the last command, and CMD+b/f to to back or forward a word. If I try the same in tmux inside the regular macos terminal it does not work (seems to give ^C). If I try it in Kitty outside of tmux it does not work (seems to give 6;9u). I can't find anything in my .tmux.conf. I really appreciate that it works but would like it to work outside of tmux in Kitty as well 😄

**Comment by kovidgoyal** [comment, marked as answer]
Presumably tmux is turning on the kitty keyboard protocol and then
mis-interpreting the resulting key codes. If you want to map cmd+.
to alt+. read the kitty faq for how to do it.

https://sw.kovidgoyal.net/kitty/faq/#how-do-i-map-key-presses-in-kitty-to-different-keys-in-the-terminal-program

**Comment by marlun** [comment]
Thanks, I'll use the mis-interpretation until I can't and use the link when I need to ☺️

**Comment by tamsky** [comment]
If anyone has a shorter implementation -- I'd love to learn it.
```
# (optional) if you want Option to be a real key, and not a UTF compose key
# macos_option_as_alt on

map cmd+a send_key meta+a
map cmd+b send_key meta+b
map cmd+c send_key meta+c
map cmd+d send_key meta+d
map cmd+e send_key meta+e
map cmd+f send_key meta+f
map cmd+g send_key meta+g
map cmd+h send_key meta+h
map cmd+i send_key meta+i
map cmd+j send_key meta+j
map cmd+k send_key meta+k
map cmd+l send_key meta+l
map cmd+m send_key meta+m
map cmd+n send_key meta+n
map cmd+o send_key meta+o
map cmd+p send_key meta+p
map cmd+q send_key meta+q
map cmd+r send_key meta+r
map cmd+s send_key meta+s
map cmd+t send_key meta+t
map cmd+u send_key meta+u
map cmd+v send_key meta+v
map cmd+w send_key meta+w
map cmd+x send_key meta+x
map cmd+y send_key meta+y
map cmd+z send_key meta+z
map cmd+0 send_key meta+0
map cmd+1 send_key meta+1
map cmd+2 send_key meta+2
map cmd+3 send_key meta+3
map cmd+4 send_key meta+4
map cmd+5 send_key meta+5
map cmd+6 send_key meta+6
map cmd+7 send_key meta+7
map cmd+8 send_key meta+8
map cmd+9 send_key meta+9
```
